docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+
+```xml
+<PackageReference Include="<package-id>" Version="" IncludeAssets="" ExcludeAssets="" PrivateAssets="" />
+```
+
+The required `Version` attribute specifies the version of the package to restore. The attribute respects the rules of the [NuGet versioning](/nuget/reference/package-versioning#version-ranges-and-wildcards) scheme. The default behavior is a minimum-version, inclusive match. For example, specifying `Version="1.2.3"` is equivalent to NuGet notation `[1.2.3, )` and means the resolved package will have the version 1.2.3 if it's available or a later version if not.
+
+The `IncludeAssets` attribute specifies which assets belonging to the package specified by `PackageReference` should be consumed. By default, all package assets are included.
+
+The `ExcludeAssets` attribute specifies which assets belonging to the package specified by `PackageReference` should not be consumed.
+
+The `PrivateAssets` attribute specifies which assets belonging to the package specified by `PackageReference` should be consumed but not flow to the next project. The `Analyzers`, `Build`, and `ContentFiles` assets are private, by default, when this attribute is not present.
+
+The :::no-loc text="assets"::: attributes can contain one or more of the following values. Separate multiple values with a semicolon `;` character.
+
+| Value | Effect |
+| - | - |
+| `Compile` | The contents of the *lib* folder are available to compile against. |
+| `Runtime` | The contents of the *runtime* folder are distributed. |
+| `ContentFiles` | The contents of the *contentfiles* folder are used. |
+| `Build` | The props/targets in the *build* folder are used. |
+| `Native` | The contents from native assets are copied to the *output* folder for runtime. |
+| `Analyzers` | The analyzers are used. |
+| `None` | None of the assets are used. (This value cannot be combined.) |
+| `All` | All assets are used. (This value cannot be combined.) |
+
+### PackageTargetFallback
+
+The `PackageTargetFallback` property lets you specify a set of compatible targets to be used when restoring packages. It's designed to allow packages that use the dotnet [TxM (Target x Moniker)](/nuget/schema/target-frameworks) to operate with packages that don't declare a dotnet TxM. If you don't add the `PackageTargetFallback` property, then all the packages your project depends on must also have a dotnet TxM.
+
+The following example specifies fallbacks for all targets:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
+The following example uses the `Condition` attribute to specify fallbacks only for the `netcoreapp2.1` target:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageTargetFallback Condition="'$(TargetFramework)'=='netcoreapp2.1'">$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

Comment:
```suggestion
    <TargetFrameworks>net48;netcoreapp2.1</TargetFrameworks>
    <PackageTargetFallback Condition="'$(TargetFramework)'=='netcoreapp2.1'">$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

I think it's worth expanding the example to explicitly multitarget since that's the only way the condition would make sense.